src/modules/drawer.js

```diff
@@ -369,20 +299,85 @@ class Drawer extends React.Component {
         }
     }
 
-    _transitionProps(el) {
-        let t;
-        let transitions = {
-            'transition': 'transitionend',
-            'oTransition': 'oTransitionEnd',
-            'MozTransition': 'transitionend',
-            'WebkitTransition': 'webkitTransitionEnd',
-        };
+    setStartOfTransform() {
+        const { positionYOffset } = this.props;
+        const translateX = this.isPositionX('left') ? TRANSLATE_X_LEFT_START : TRANSLATE_X_RIGHT_START;
 
-        for (t in transitions) {
-            if (el.style[t] !== undefined) {
-                return transitions[t];
-            }
+        this.drawerContainerRef.style.transform = _.isNumber(positionYOffset) ?
+            `${translateX} translateY(${positionYOffset}px)` :
+            translateX;
+    }
+
+    isPositionX(position) {
+        const { positionX } = this.props;
+
+        return positionX === position;
+    }
+
+    isPositionY(position) {
+        const { positionY } = this.props;
+
+        return positionY === position;
+    }
+
+    render() {
+        const {
+            children,
+            className,
+            positionYOffset,
+            wing,
+        } = this.props;
+        const { isOpen } = this.state;
+
+        if (!isOpen) {
+            return false;
         }
+
+        const drawerClasses = ClassNames('ui', 'drawer', className, {
+            'left-position': this.isPositionX('left'),
+            'top-position': this.isPositionY('top'),
+            'bottom-position': this.isPositionY('bottom'),
+        });
+
+        return (
+            <Portal>
+                <div
+                    className={drawerClasses}
+                    ref={(ref) => { this.drawerRef = ref; }}
+                >
+                    <div
+                        className="drawer-container"
+                        ref={(ref) => { this.drawerContainerRef = ref; }}
+                        style={{
+                            height: _.isNumber(positionYOffset) ? `calc(100% - ${positionYOffset}px)` : null,
+                            // transform: this.setStartOfTransform(),
```

Comment:
Is there a good reason to keep this commented out?

Reply:
removed.